### DISCUSSION TITLE
Adding new languages in Weblate

### DISCUSSION THREAD

**Discussion by sirviot** [lxqt/lxqt#1876] [Q&A] [answered]
I tried to update translations using the Weblate, but although I was able to add a new language for a component, I wasn't able to save or suggest any translations for it. I also wasn't able to save or suggest any translations for existing languages.

Is there somekind of an acceptance period before logged in user can edit translations?

**Comment by stefonarch** [comment]
For some reason I cannot aswer but only suggest an answer (first day of discussion feature here!)

There shouldn't be restrictions atm, we're [discussing](https://github.com/lxqt/lxqt/issues/1495#issuecomment-741642632) it just now.
But thanks, one of my tests yesterday produced this I think, retry now please.

**Reply by stefonarch** [reply]
I changed the title to be more precise.

**Comment by stefonarch** [comment, marked as answer]
For adding  language files that are not already present  [contacting  administrators](https://translate.lxqt-project.org/contact/) is needed, permissions will be upgraded for asking users.
Upload for existing files should work for all users.

**Comment by sirviot** [comment]
I tried again today, and it worked great! I added Finnish translation for `lxqt-powermanagement` and `lxqt-powermanagement-battery-info`, and the next step - if I understood the process correctly - is to sit down and wait for new builds/releases to happen.

**Reply by stefonarch** [reply]
Right, an PR with an empty template has arrived just now, usually it takes ~24h for all.
